grid.cpp

```diff
@@ -211,8 +215,23 @@ void initializeGrids(
       // Each initialization has to be independent to avoid threading problems 
 
       // Allow the project to set up data structures for it's setCell calls
-      project.setupBeforeSetCell(cells);
-      
+      bool needCurl=false;
+      project.setupBeforeSetCell(cells, mpiGrid, needCurl);
+      if (needCurl==true) {
+	 // Communicate the perturbed B-fields read from the start file over to FSgrid
+	 technicalGrid.updateGhostCells(); // This needs to be done at some point
```

Comment:
Should this update not be pulled out of the `if`? At least we did something like that in the AMR filtering branch.